include/unifex/sync_wait.hpp

```diff
@@ -188,7 +188,7 @@ namespace _sync_wait_cpo {
 
         auto operation = connect(
           (Sender&&)sender,
-          _sync_wait::thread_unsafe_receiver<Result, StopToken&&>{
+          _sync_wait::thread_unsafe_receiver<Result, StopToken>{
```

Comment:
Why was this change necessary?

Reply:
&& did not look like it was doing anything.

Reply:
It was avoiding a copy of the StopToken and instead capturing a reference to the stop-token passed to sync_wait.

With this change it will still capture a reference to the stop-token in the receiver if the caller passed an lvalue, but will now move-construct the stop-token into the receiver if the caller passed an rvalue.

If we think that we should be capturing a copy of the stop-token in the receiver instead of a reference then perhaps we should be decay-copying the stop-token here instead?

I guess this is all kind of moot after PR #80 is merged anyway.